### DISCUSSION TITLE
Can anyone Pls. confim on which OS Version the script worx

### DISCUSSION THREAD

**Discussion by witzker** [ej52/proxmox-scripts#133] [General]
Hi, as I have problems with my old VM of NPM I tried to set up new with this script a fresh Alpine 3.18
ERROR.

Boot problem #130
 Script does not work on Alpine 3.18.2 #131 

As NRP is an essential part of my lab, I need to get it up and running asap.

Therefor, I would kindly ask on which version of
Alpine, Debian and Ubuntu the script worked for you.
Many THX for saving time at
Trial and Error.

Your experience can then be put into the README
 Pls. add version of Alpine, Debian and Ubuntu on which it's working. #132

**Comment by witzker** [comment]
I found This:
https://openresty.org/en/linux-packages.html
![grafik](https://github.com/ej52/proxmox-scripts/assets/4120266/33c082ab-932c-47e1-bc7e-4de0812367bb)

downloaded:
![grafik](https://github.com/ej52/proxmox-scripts/assets/4120266/d2db77e0-4317-4668-9bef-f0352f2529dd)

Started the script
Result:
![grafik](https://github.com/ej52/proxmox-scripts/assets/4120266/2a00c687-75ee-4bbd-a8b0-638d93190ffe)

Pls Help to get this installed

**Comment by ej52** [comment]
@witzker OS Version support added to README
